dataland-frontend/tsconfig.cypress-ct.json

```diff
@@ -0,0 +1,15 @@
+// Config file for Cypress component tests
+{
+  "extends": "./tsconfig.app.json",
+  "include": [
+    "src/**/*",
+    "src/**/*.vue",
+    "build/clients/**/*",
+    "tests/component/**/*"
+  ],
+  "exclude": [],
```

Comment:
Kann die Zeile weg? Weil in der cypress-e2e.json gibt es auch keinen expliziten `exclude`